superset/assets/javascripts/chart/Chart.jsx

```diff
@@ -188,6 +188,7 @@ class Chart extends React.PureComponent {
       });
       this.props.actions.chartRenderingSucceeded(this.props.chartKey);
     } catch (e) {
+      console.error(e);  // eslint-disable-line
```

Comment:
remove this console.error()

Reply:
I was thinking about keeping that one around (that's why I silenced the linter). This gives us a stack trace instead of swallowing errors happening in the visualizations.